test/extensions/filters/listener/original_src/original_src_fuzz_test.cc

```diff
@@ -0,0 +1,47 @@
+#include "envoy/common/exception.h"
+#include "envoy/extensions/filters/listener/original_src/v3/original_src.pb.h"
+
+#include "common/network/utility.h"
+
+#include "extensions/filters/listener/original_src/original_src.h"
+
+#include "test/extensions/filters/listener/original_src/original_src_fuzz_test.pb.validate.h"
+#include "test/fuzz/fuzz_runner.h"
+#include "test/mocks/network/mocks.h"
```

Comment:
nit: can you double-check to make sure that the dependencies you have here are just the direct set of dependencies listed in the bazel build rule?

Reply:
Yeah, decided to change the BUILD file to reflect the includes more directly.